src/scripts/downloadUiAssets.js

```diff
@@ -322,12 +388,32 @@ async function main() {
   console.log("Downloading from", url);
 
   try {
-    // Check versions compatibility
-    await exports.checkCompatibility(await getVersions(url), url);
-    // Get Workspace Web Edition archive
-    await getArchive(url + 'ui/wwe/', './ui-assets/wwe');
-    // Get GWS Auth UI archive
-    await getArchive(url + 'ui/auth/', './ui-assets/auth');
+    let isAzurePlatform;   
+    isAzurePlatform = await checkUrls(url);
+    console.log('result profile', isAzurePlatform);
```

Comment:
Do you plan to expose this trace to customer?

Reply:
Deleted this console.log